recipes/libcap/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibcapConan(ConanFile):
+    name = "libcap"
+    license = ("GPL-2.0-only", "BSD-3-Clause")
```

Comment:
It should be `GPL-2.0-or-later`
See https://git.kernel.org/pub/scm/libs/libcap/libcap.git/tree/License#n345

Reply:
This is a default paragraph of the GPL 2.0 license and explains how to use the license. It isn't a statement by the project maintainer if a later version can be picked or not

See https://spdx.org/licenses/GPL-2.0-only.html#licenseText for a comparision

Reply:
Oh, thanks for explaining

Reply:
The difference between those license is very small:

GPL-2.0-only:

This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; version 2.

GPL-2.0-or-later:

This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; **either version 2 of the License, or (at your option) any later version.**

A single phrase makes all the difference.

The libcap's license contains "either version 2 of the License, or (at your option) any later version", so it's GPL-2.0-or-later.

Reply:
`GPL-2.0-only` and `GPL-2.0-or-later` are variants on the same license. And full texts of licenses are identical (including "**either version 2 of the License, or (at your option) any later version**" in the "**How to Apply These Terms to Your New Programs**" section). The difference is only in license notices that should be included into the heading of the project files (it can be with or without "**or any later version**" statement).
`libcap` source files doesn't have standard GPL license headers. But the License file itself says: "**this product may be distributed under the terms of the GNU General Public License (v2.0 - see below)**" without mention of later version of GPL. So it should be `GPL-2.0-only`.
Also `libcap` packages from Buildroot and Debian projects are mentioning license as `GPL-2.0-only`.
Just to be sure, I asked `libcap` maintainer (Andrew G Morgan) by email about license. He replied that `GPL-2.0-only` is a correct SPDX identifier.
I'm almost sure that we shouldn't change license in the recipe.